lib/galaxy/tool_util/deps/container_resolvers/mulled.py

```diff
@@ -688,7 +688,7 @@ def resolve(self, enabled_container_types, tool_info, install=False, **kwds):
         if len(targets) == 0:
             return None
         if self.auto_install or install:
-            mull_targets(targets, involucro_context=self.involucro_context, **self._mulled_kwds)
+            mull_targets(targets, involucro_context=self.involucro_context, namespace=self.namespace, **self._mulled_kwds)
```

Comment:
Isn't `namespace` already passed as part of `**self._mulled_kwds` ?

Reply:
Huh, indeed. Let me check.

Reply:
Seems that I have been on the wrong path here for some reason the `TestSingularityJobsIntegration` tests in `test/integration/test_containerized_jobs.py` seem to create `quay.io/biocontainers/mulled-v2-8186960447c5cb2faa697666dc1e6d919ad23f3e` which tricked me.

Will try to find out why ..